### PR TITLE
Update testfiles in webos-js sample

### PR DESCRIPTION
**Pull request by gouniLee**
* testResources.js : Test cases are showing the same result when running in `localize` and `generate` modes
* testResourceLocalize.js: Special test case for normal `localize` mode 
* testResourceGenerate.js: Special test case for normal `generate` mode

### CODE REVIEW

File: webos-js/package.json [hunk 1]
```diff
@@ -7,10 +7,11 @@
...
+        "loc-generate": "loctool generate -2 -x xliffs --projectType custom --sourceLocale en-KR --resourceFileTypes json=webos-json-resource --plugins webos-javascript,webos-appinfo-json  --projectId sample-webos-js -l de-DE,en-AU,en-US,en-GB,en-JP,es-ES,es-CO,fr-CA,fr-FR,ja-JP,ko-KR,ko-US --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB,en-JP:en-GB",
+        "loc-generate-debug": "node --inspect-brk node_modules/loctool/loctool.js generate -2 -x xliffs --projectType custom --sourceLocale en-KR --resourceFileTypes json=webos-json-resource --plugins webos-javascript,webos-appinfo-json --projectId sample-webos-js -l de-DE,en-AU,en-US,en-GB,en-JP,es-ES,es-CO,fr-CA,fr-FR,ja-JP,ko-KR,ko-US --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB,en-JP,en-GB",
         "clean": "rm -rf *.xliff resources",
-        "test": "node test/testResources.js"
+        "test": "npm run clean;npm run loc;node test/testResources.js; node test/testResourceLocalize.js",
+        "test-generate": "npm run clean;npm run loc-generate;node test/testResources.js; node test/testResourceGenerate.js"
```
Comment (ehoogerbeets) [inline webos-js/package.json:14]: Should consider using the "npm-run-all" package:

`"test-generate": "npm-run-all clean loc-generate test"`

**Review by ehoogerbeets — APPROVED** [review]

File: webos-js/package.json (hunk 1 above)
Comment (gouniLee) [reply webos-js/package.json]: oh, That's nice to know! I update it.

File: webos-js/package.json [hunk 2]
```diff
@@ -7,17 +7,22 @@
...
         "loc-generate-debug": "node --inspect-brk node_modules/loctool/loctool.js generate -2 -x xliffs --projectType custom --sourceLocale en-KR --resourceFileTypes json=webos-json-resource --plugins webos-javascript,webos-appinfo-json --projectId sample-webos-js -l de-DE,en-AU,en-US,en-GB,en-JP,es-ES,es-CO,fr-CA,fr-FR,ja-JP,ko-KR,ko-US --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB,en-JP,en-GB",
-        "clean": "rm -rf *.xliff resources",
-        "test": "npm run clean;npm run loc;node test/testResources.js; node test/testResourceLocalize.js",
-        "test-generate": "npm run clean;npm run loc-generate;node test/testResources.js; node test/testResourceGenerate.js"
+        "test": "node test/testResources.js; node test/testResourceLocalize.js",
+        "test-generate": "node test/testResources.js; node test/testResourceGenerate.js",
+        "test:all": "npm-run-all clean loc-generate test-generate",
```
Comment (ehoogerbeets) [inline webos-js/package.json:15]: Should this be:

```
"test:all": "npm-run-all clean loc-generate test",
```

???

Comment (gouniLee) [reply webos-js/package.json]: ah, yes, it should be that. ^^; I fixed it. Thanks!

**Pull request merged**